From 6bedec045374a0e841dc8eedcd4ffcc9345481dc Mon Sep 17 00:00:00 2001
From: Steven Barrett <steven@liquorix.net>
Date: Mon, 11 Jul 2022 19:10:30 -0500
Subject: [PATCH] ZEN: cpufreq: Remove schedutil dependency on
 Intel/AMD P-State drivers

Although both P-State drivers depend on schedutil in Kconfig, both code
bases do not use any schedutil code.  This arbitrarily enables schedutil
when unwanted in some configurations.
---
 drivers/cpufreq/Kconfig.x86 | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/cpufreq/Kconfig.x86 b/drivers/cpufreq/Kconfig.x86
index d6190b8dde4b99..9d10fde562c71f 100644
--- a/drivers/cpufreq/Kconfig.x86
+++ b/drivers/cpufreq/Kconfig.x86
@@ -8,7 +8,6 @@ config X86_INTEL_PSTATE
 	select ACPI_PROCESSOR if ACPI
 	select ACPI_CPPC_LIB if X86_64 && ACPI && SCHED_MC_PRIO
 	select CPU_FREQ_GOV_PERFORMANCE
-	select CPU_FREQ_GOV_SCHEDUTIL if SMP
 	help
 	  This driver provides a P state for Intel core processors.
 	  The driver implements an internal governor and will become
@@ -38,7 +37,6 @@ config X86_AMD_PSTATE
 	depends on ACPI
 	select ACPI_PROCESSOR
 	select ACPI_CPPC_LIB if X86_64
-	select CPU_FREQ_GOV_SCHEDUTIL if SMP
 	select ACPI_PLATFORM_PROFILE
 	select POWER_SUPPLY
 	help

